rnd/autogpt_builder/src/components/CustomNode.tsx

```diff
@@ -495,10 +495,11 @@ export function CustomNode({ data, id, width, height }: NodeProps<CustomNode>) {
         )}
       </div>
       <InputModalComponent
+        title={activeKey ? `Enter ${beautifyString(activeKey)}` : undefined}
         isOpen={isModalOpen}
         onClose={() => setIsModalOpen(false)}
         onSave={handleModalSave}
-        value={inputModalValue}
+        initialValue={inputModalValue}
```

Comment:
nit: why not `defaultValue`
https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement#defaultvalue

Reply:
thanks, renamed